### PR TITLE
update columns

### PR DESCRIPTION
**Pull request by J22Melody**
@bambooforest Indeed they updated a few columns, one thing special is that they replace the former "ID" column with a "voiceless, cluster" column.

### CODE REVIEW

File: cldfbench_tonodb.py [hunk 1]
```diff
@@ -109,6 +109,7 @@ def cmd_makecldf(self, args):
             tone_id = hashlib.md5(tone.encode('utf8')).hexdigest().upper()
 
             # values.csv
+            row['ID'] = row['voiceless, cluster']
```
Comment (J22Melody) [inline cldfbench_tonodb.py]: see here the replacement

File: cldfbench_tonodb.py [hunk 2]
```diff
@@ -31,7 +31,7 @@ class Dataset(BaseDataset):
     dir = pathlib.Path(__file__).parent
     id = "tonodb"
     # valueTableProperties = ['LanguageVariety', 'TriggeringContext' , 'Extra', 'Height', 'Contour', 'Phonation', 'ToneDescription', 'ChaoNumerals', 'Notes', 'EffectOnPitch', 'ResultantSystem', 'Type', 'Onset', 'Coda', 'Stress/quantity', 'Wordtype', 'long vowel']
-    valueTableProperties = ['LanguageVariety', 'TriggeringContext', 'Extra', 'Height', 'Contour', 'Phonation', 'ToneDescription' , 'ChaoNumerals', 'Notes', 'EffectOnPitch', 'ResultantSystem', 'Type', 'Onset', 'Coda', 'Stress/quantity', 'Wordtype', 'Nucleus']
+    valueTableProperties = ['LanguageVariety', 'TriggeringContext', 'Extra', 'Height', 'Contour', 'Phonation', 'ToneDescription' , 'ChaoNumerals', 'Notes', 'EffectOnPitch', 'ResultantSystem', 'Type', 'Onset', 'OnsetManner', 'OnsetVoicing', 'OnsetAspiration', 'Coda', 'CodaPhonation', 'CodaGlottal', 'CodaManner', 'Stress/quantity', 'NucleusATR', 'NucleusLength', 'NucleusHight', 'Wordtype', 'Nucleus']
```
Comment (J22Melody) [inline cldfbench_tonodb.py]: new columns

**Comment by bambooforest** [comment]
@J22Melody -- ugh, that's bad. And my bad for missing it. I've updated the column back to `ID`. Also corrected a misspelling in NucleusHight -> NucleusHeight.

Thanks for catching this.

**Comment by J22Melody** [comment]
Great! Updated again accordingly.

**Pull request merged**